.github/workflows/linters.yml

```diff
@@ -77,3 +77,11 @@ jobs:
         run: |
           make lint-py-build || true
           NODE=$(which node) make lint-py
+
+  lint-codeowners:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: mszostok/codeowners-validator@v0.4.0
```

Comment:
Do we have a policy on using third-party actions?

Reply:
Not that I'm aware of

Reply:
We definitely have it enabled because this action ran